code/modules/reagents/reagent_containers/inhaler.dm

```diff
@@ -103,6 +103,14 @@
 		to_chat(user,"<span class='notice'>The reagents inside \the [src] are already secured.</span>")
 	return
 
+/obj/item/weapon/reagent_containers/inhaler/attackby(obj/item/weapon/W, mob/user)
+	if(W.isscrewdriver() && !is_open_container())
+		to_chat(user,"<span class='notice'>Using \the [W], you unsecure the inhaler's lid.</span>") // it locks shut after being secured
+		flags |= OPENCONTAINER
```

Comment:
Same here as in the hypospray. I'm not seeing a close action being added.

Reply:
It's in attack_self, the close was already there.